tests/tabs/test_recently_closed_tabs.py

```diff
@@ -0,0 +1,55 @@
+import logging
+
+from selenium.webdriver import Firefox
+from selenium.webdriver.support import expected_conditions as EC
+
+from modules.browser_object import PanelUi, TabBar
+
+links = [
```

Comment:
I might consider only using sites where you know the load latency is very small. If you can confirm that is the case for all of these, go for it. If not, you can open a bunch of about pages instead.